storage/src/test/java/tech/pegasys/artemis/storage/MapDbDatabaseTest.java

```diff
@@ -80,6 +81,11 @@
 
         @Override
         public void onGenesis(Store store) {}
+
+        @Override
+        public SafeFuture<Optional<Store>> onStoreRequest() {
+          return null;
```

Comment:
Should return a `failedFuture` instead of `null`.  While it's not called today getting a null future returned is a nasty surprise.

Reply:
Could this extend `StubStorageUpdateChannel` and just override `onStorageUpdate`?

Reply:
Yeah definitely, that's a great idea.

Reply:
Done.